cmd/earthly/satellite_cmds.go

```diff
@@ -310,6 +312,14 @@ func (app *earthlyApp) actionSatelliteLaunch(cliCtx *cli.Context) error {
 		return err
 	}
 
+	if cliCtx.IsSet(satelliteSizeFlag) {
+		if !cloud.ValidSatelliteSize(size) {
+			return errors.Errorf("not a valid size: %s", size)
+		}
+	} else {
+		size = cloud.DefaultSatelliteSize
```

Comment:
Could this be defined via the default `Value` `StringFlag` field instead?

Reply:
Yes, for sure. Wasn't aware of that feature.

Reply:
Added this.